### PR TITLE
create documentation for the new liquid template unit testing tool

### PR DESCRIPTION
**Pull request by holdenhinkle**
Resolves [ticket #21523](https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/21523) - Add documentation for the new Liquid Template Unit Testing Tool to `va.gov-team` repo

### CODE REVIEW

**Comment by holdenhinkle** [comment]
@mpelzsherman Do you have some time to write up some "dos and don'ts" for the liquid template unit testing tool?

**Review by timwright12 — APPROVED** [review]
wonderful, thank you!

File: platform/testing/liquid-templates/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Liquid Template Unit Testing Framework
+
+The Liquid Template Unit Testing Framework was created to replace Cypress for unit testing the logic in `liquid` templates because Cypress is slow, heavy, and is overkill for this purpose. More importantly, this framework provides us with total control over the test data which is critical for testing `liquid` templates, something that Cypress does not provide. For example, a you can test a given `liquid` template against any number of fixtures that represent different enteries into the Drupal CMS, so bugs in `liquid` templates can be discovered more quickly.
+
```
Comment (mpelzsherman) [inline platform/testing/liquid-templates/README.md:4]: Found a typo: "enteries" s/b "entries"

Comment (holdenhinkle) [reply platform/testing/liquid-templates/README.md:4]: Thanks. Fixed.

**Pull request merged**